src/TimeZoneNames/TimeZoneNames.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Provides localized time zone names.</Description>
     <Authors>Matt Johnson-Pint</Authors>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net48</TargetFrameworks>
```

Comment:
I'll be bumping to 4.6.2, but we can't go all the way to 4.8.

Reply:
I changed it to 4.6.2.  I normally wouldn't make this change in this type of PR, but 4.6.1 won't build on my machine.